tools/ci-build/publisher/src/subcommand/publish.rs

```diff
@@ -162,21 +172,26 @@ pub async fn correct_owner(handle: &PackageHandle, category: &PackageCategory) -
                 // Teams in crates.io start with `github:` while individuals are just the GitHub user name
                 added_individual |= !crate_owner.starts_with("github:");
             }
-            for incorrect_owner in incorrect_owners {
+            for crate_owner in owners_to_be_removed {
+                // Trying to remove them will result in an error due to a bug in crates.io
+                // Upstream tracking issue: https://github.com/rust-lang/crates.io/issues/2736
+                if crate_owner == SMITHY_RS_SERVER_OWNER || crate_owner == RUST_SDK_OWNER {
+                    continue;
+                }
                 // Adding an individual owner requires accepting an invite, so don't attempt to remove
                 // anyone if an owner was added, as removing the last individual owner may break.
                 // The next publish run will remove the incorrect owner.
                 if !added_individual {
-                    cargo::RemoveOwner::new(&handle.name, incorrect_owner)
+                    cargo::RemoveOwner::new(&handle.name, crate_owner)
                         .spawn()
                         .await
-                        .context(format!("remove incorrect owner `{}` from crate `{}`", incorrect_owner, handle))?;
```

Comment:
I assume this call failed in [this run](https://github.com/awslabs/smithy-rs/actions/runs/4106453596/jobs/7085205295#step:8:364). Why didn't we get this error message in the logs?

Reply:
Also you may want `.with_context` instead to avoid allocating the string.

Reply:
Done in https://github.com/awslabs/smithy-rs/pull/2318/commits/28ee6f3c7ae7fc53818439c2badcb6042469ec8d
I'm looking into the logs 🤔

Reply:
We got the log for each attempt (`2023-02-06T18:24:58.190589Z  INFO publisher::retry: Correcting ownership of `aws-smithy-async` failed on attempt 1 with retryable error: remove incorrect owner `github:awslabs:smithy-rs-server` from crate `aws-smithy-async-0.54.2`. Will retry after 5s`), but it lacks the inner details. I have fixed it in https://github.com/awslabs/smithy-rs/pull/2318/commits/fa9afc72632cd33eedb1e44750e6fd391a3f7462

Reply:
Great ⭐